lib/pages/trending/trending.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:get/get_state_manager/get_state_manager.dart';
+import 'package:soundspace/constants/controllers.dart';
+import 'package:soundspace/helpers/responsiveness.dart';
+import 'package:soundspace/pages/trending/widgets/trending_cards_medium.dart';
+import 'package:soundspace/pages/trending/widgets/trending_cards_small.dart';
+import 'widgets/trending_cards_large.dart';
+
+class TrendingPage extends StatelessWidget {
+  const TrendingPage({Key? key}) : super(key: key);
```

Comment:
I do not think we actually need to define a key for a stateless widget I could be wrong but I dont see a purpose for doing it.